bitcash/format.py

```diff
@@ -35,10 +43,10 @@ def verify_sig(signature, data, public_key):
     return _vs(signature, data, public_key)
 
 
-def address_to_public_key_hash(address):
+def address_to_public_key_hash(address, regtest=False):
     # LEGACYADDRESSDEPRECATION
     # FIXME: This legacy address support will be removed.
-    address = cashaddress.to_cash_address(address)
+    address = cashaddress.to_cash_address(address, regtest)
```

Comment:
This doesn't make sense if it can detect regtest further down.

Reply:
The regtest flag needs to be passed to the `to_cash_address` function for the reasons mentioned in
[https://github.com/pybitcash/bitcash/pull/82#discussion_r509907536](https://github.com/pybitcash/bitcash/pull/82#discussion_r509907536)

Reply:
['Regtest' flag removed along side legacy address support.](https://github.com/pybitcash/bitcash/pull/82/commits/e90ce7868f4a091759cb3f8970274c6771a27fec)